dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/TracingInterceptorTest.kt

```diff
@@ -362,13 +362,36 @@ internal open class TracingInterceptorTest {
         val response = testedInterceptor.intercept(mockChain)
 
         verify(mockSpanBuilder).withOrigin(getExpectedOrigin())
+        verify(mockSpan as MutableSpan).resourceName = fakeUrl.substringBefore("?")
```

Comment:
Here you're duplicating the logic, making your test useless. Because the algorithm written is the same, it's bound to be green, and yet it doesn't test what you want it to test. Instead you should do something like : 

```
    val baseUrl: String = … // a random url with protocol, host and path, e.g.: https://www.example.com/foo/bar/spam
    val queryParams: Map<String,String> = … // a random map of qp
    val queryParamString = queryParams.joinToString("&") { "${it.key}=${it.value}" }
    val url = "$baseUrl?$queryParamString"
```